src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs

```diff
@@ -376,6 +377,7 @@ private Task WriteHexInt32Async(int value, bool async)
             return WriteAsciiStringAsync(value.ToString("X", CultureInfo.InvariantCulture), async);
         }
 
+        [UnsupportedOSPlatform("tvos")]
```

Comment:
I'm not sure about this. Does it mean that SocketHttpHandler is unusable on tvos?

Reply:
Yeah, that does not sound right. I suppose we may need a dummy reimplementation of `AuthenticationHelper.NtAuth.cs` that does something like this:
```csharp
namespace System.Net.Http
{
    internal static partial class AuthenticationHelper
    {
        public static Task<HttpResponseMessage> SendWithNtProxyAuthAsync(HttpRequestMessage request, Uri proxyUri, bool async, ICredentials proxyCredentials, HttpConnection connection, HttpConnectionPool connectionPool, CancellationToken cancellationToken)
        {
            return connection.SendAsyncCore(request, async, cancellationToken);
        }

        public static Task<HttpResponseMessage> SendWithNtConnectionAuthAsync(HttpRequestMessage request, bool async, ICredentials credentials, HttpConnection connection, HttpConnectionPool connectionPool, CancellationToken cancellationToken)
        {
            Debug.Assert(request.RequestUri != null);
            return connection.SendAsyncCore(request, async, cancellationToken);
        }
    }
}
```